tools-local/tasks/installer.tasks/CopyNupkgAndChangeVersion.cs

```diff
@@ -0,0 +1,148 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Build.Framework;
+using Newtonsoft.Json.Linq;
+using NuGet.Versioning;
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class CopyNupkgAndChangeVersion : BuildTask
```

Comment:
Why is this needed? I don't see you including any usage of it in your PR?

Reply:
The usage is already checked in https://github.com/dotnet/runtime/blob/release/5.0/src/installer/tests/PrepareTestAssets/PrepareTestAssets.proj#L129
We are not hitting this error in master because this target is only executed while producing stable bits.